include/cantera/oneD/Flow1D.h

```diff
@@ -548,15 +557,15 @@ class Flow1D : public Domain1D
                               double rdt, size_t jmin, size_t jmax);
 
     /**
-     * Evaluate the lambda equation residual.
+     * Evaluate the radial pressure gradient equation residual.
      *
      * @f[
      *    \frac{d\Lambda}{dz} = 0
      * @f]
      *
-     * The lambda equation serves as an eigenvalue that allows the momentum
+     * The Lambda equation serves as an eigenvalue that allows the momentum
      * equation and continuity equations to be simultaneously satisfied in
-     * axisymmetric flows. The lambda equation propagates information from
+     * axisymmetric flows. The Lambda equation propagates information from
```

Comment:
While we're cleaning this up:
```suggestion
     * The radial pressure gradient @f$ \Lambda @f$ serves as an eigenvalue that allows
     * the momentum and continuity equations to be simultaneously satisfied in
     * axisymmetric flows. This equation propagates information from
```